scripts/servers.py

```diff
@@ -699,6 +706,11 @@ def managed_webdriverio_server(
     if chrome_version is None:
         chrome_version = get_chrome_verison()
 
+    if mobile:
+        os.environ['MOBILE'] = 'true'
+    else:
+        os.environ['MOBILE'] = 'false'
```

Comment:
Please make sure these get reverted after the tests complete, even if the tests fail partway through (the standard way to do this is with a context manager). We don't want to permanently change the user's environment to avoid causing side-effects

Reply:
I have used env variables with context manager to so this.